integration/copy_test.go

```diff
@@ -79,6 +79,36 @@ func (s *CopySuite) TearDownSuite(c *check.C) {
 	}
 }
 
+/*
+TestRunShell is not really a test; it is a convenient way to use the registry setup code
+in openshift.go and CopySuite to get an interactive environment for experimentation.
+
+To use it, uncomment the “test” below, and run:
```

Comment:
It could be; but really commenting out the code is pretty cumbersome (we also have to comment out an import), so I have rewritten this to use a build tag instead.